lib/puppet/parser/ast/pops_bridge.rb

```diff
@@ -58,7 +58,7 @@ def children
 
   class ExpressionSupportingReturn < Expression
     def initialize args
-      super
+      super(**args)
     end
```

Comment:
Since we're just calling the super, I think we can drop this `initialize` method.